test/wdio.8.mjs

```diff
@@ -0,0 +1,463 @@
+        babelOpts: {
+            ignore: []
+        },
+    },
+
+    // ============
+    // Capabilities
+    // ============
+    // Define your capabilities here. WebdriverIO can run multiple capabilities at
+    // the same time. Depending on the number of capabilities, WebdriverIO
+    // launches several test sessions. Within your `capabilities`, you can
+    // overwrite the `spec` and `exclude` options in order to group specific specs
+    // to a specific capability.
+
+    // First, you can define how many instances should be started at the same
+    // time. Let's say you have 3 different capabilities (Chrome, Firefox, and
+    // Safari) and you have set `maxInstances` to 1. wdio will spawn 3 processes.
+    //
+    // Therefore, if you have 10 spec files and you set `maxInstances` to 10, all
+    // spec files will be tested at the same time and 30 processes will be
+    // spawned.
+    //
+    // The property handles how many capabilities from the same test should run
+    // tests.
+    maxInstances: parseInt(process.env.PARALLEL, 10) || 1,
+    // Or set a limit to run tests with a specific capability.
+    maxInstancesPerCapability: 10,
+
+    // Inserts WebdriverIO's globals (e.g. `browser`, `$` and `$$`) into the
+    // global environment. If you set to `false`, you should import from
+    // `@wdio/globals`. Note: WebdriverIO doesn't handle injection of test
+    // framework specific globals.
+    injectGlobals: true,
+
+    // Additional list of node arguments to use when starting child processes.
+    execArgv: [],
+
+    // If you have trouble getting all important capabilities together, check out the
+    // BrowserStack platform configurator - a great tool to configure your capabilities:
+    // https://www.browserstack.com/automate/capabilities?tag=selenium-4.
+    capabilities: filterCaps([
+      // Internet Explorer.
+      {
+        'browserName': 'IE',
+        'browserVersion': '11.0',
+        'bstack:options': {
+          'os': 'Windows',
+          'osVersion': '10',
+          'sendKeys': true,
+        },
+      },
+      {
+        'browserName': 'IE',
+        'browserVersion': '10.0',
+        'bstack:options': {
+          'os': 'Windows',
+          'osVersion': '8',
+          'sendKeys': true,
+          'seleniumVersion': '3.5.2',
+        },
+      },
+      {
+        'browserName': 'IE',
+        'browserVersion': '9.0',
+        'bstack:options': {
+          'os': 'Windows',
+          'osVersion': '7',
+          'sendKeys': true,
+          'seleniumVersion': '3.5.2',
+        },
+      },
+      {
+        'browserName': 'IE',
+        'browserVersion': '8.0',
+        'bstack:options': {
+          'os': 'Windows',
+          'osVersion': '7',
+          'sendKeys': true,
+          'seleniumVersion': '3.5.2',
+        },
+      },
+
+      // Android
+      // Reminder set for 16-01-2023 to uncomment this next block and check if
```

Comment:
(This date is in the past!)

Reply:
I'm going to tidy up these capabilities in another pass.